src/selectize.js

```diff
@@ -1772,9 +1772,13 @@ $.extend(Selectize.prototype, {
 
 		if (self.settings.mode === 'single' && self.items.length) {
 			self.hideInput();
-			setTimeout(function() {
+
+			// Do not trigger blur while inside a blur event,
+			// this fixes some weird tabbing behavior in FF and IE.
+			// See #1164
+			if (self.ignoreFocus) {
```

Comment:
I've used `ignoreFocus` as it's set in the `onBlur` method, maybe I should use a different, new, boolean flag?

Reply:
I'm not actively developing the project, I'll have to take your word for it